include/swift/SIL/SILInstruction.h

```diff
@@ -527,8 +527,11 @@ class SILInstruction
 public:
   using OperandValueRange =
       OptionalTransformRange<ArrayRef<Operand>, OperandToValue>;
-  OperandValueRange
-  getOperandValues(bool skipTypeDependentOperands = false) const;
+  OperandValueRange getOperandValues(
```

Comment:
I am hesitant about this part of the change. My gut feeling says that the right thing to do here is to not put this upon all users of this code. Maybe the compiler will optimize it all away... I don't know. Why not instead add a composition type: make a transform range with this property at your local use site.

Reply:
I added a new overload of `getOperandValues` without default args, and created a new `TransformedOperandValueRange`.